frontend/src/pages/Home.tsx

```diff
@@ -0,0 +1,180 @@
+import blackGoLogo from "../assets/blackGoLogo.png";
+import meetupLogo from "../assets/meetupLogo.png";
+import xLogo from "../assets/xLogo.png";
+import waves from "../assets/waves.svg";
+import speakersImage from "../assets/speakersImage.png";
+import phoneWave from "../assets/phoneWave.svg";
+
+// function Nav() {
+//   return (
+//     <nav className={"p-3 flex flex-row items-center justify-start"}>
+//       <a href="/" className={"text-primary border-b-2 border-primary p-2"}>
+//         Home
+//       </a>
+//     </nav>
+//   );
+// }
+
+function Sponsors() {
```

Comment:
can we have a components folder and move these components in there so they can be reused in the other pages?

Reply:
alright @spankie lemmi work that

Reply:
@nutcas3 Ive already pushed the changes inline with the request..should I revert them?